pnpm-lock.yaml

```diff
@@ -7292,6 +7310,43 @@ packages:
       webpack: 4.46.0
     dev: true
 
+  /@storybook/core-client@6.5.16(react-dom@17.0.2)(react@17.0.2)(typescript@5.0.3)(webpack@5.75.0):
```

Comment:
These changes look like from old diff. Could you revert lock file and install again?